nemo_skills/prompt/config/deepseek/minif2f.yaml

```diff
@@ -12,21 +12,27 @@
 # See the License for the specific language governing permissions and
```

Comment:
Let's remove this file (we don't need it, right?) and bring back example-local.yaml (somehow it says it's renamed from it)